L1Trigger/L1THGCal/plugins/veryfrontend/HGCalVFEProcessorSums.cc

```diff
@@ -83,8 +83,12 @@ void HGCalVFEProcessorSums::run(const HGCalDigiCollection& digiColl,
   for (const auto& [tc_id, tc_value] : tc_payload) {
     if (tc_value > 0) {
       const auto& [tc_compressed_code, tc_compressed_value] = tc_compressed_payload[tc_id];
+      if (tc_compressed_value > 0xFFFFFFFF)
```

Comment:
Here and below: I would rather use `std::numeric_limits`, in particular in cases where we are converting to `int` and not `uint32_t`. It is also more explicit.

Reply:
Oh yes, you are right, thanks! I have just implemented that.